example_project/mappers/Baseline.rb

```diff
@@ -1340,6 +1340,8 @@ def time_mapping(time)
               OpenStudio::Extension.set_measure_argument(osw, 'create_typical_building_from_model', 'system_type', system_type, 'create_typical_building_from_model 2')
             end
 
+            OpenStudio::Extension.set_measure_argument(osw, 'export_time_series_modelica', '__SKIP__', false)
```

Comment:
This is a good change, to now match the convention of everything skipping by default and you have to intentionally turn it on when you want it. 👍